pubspec.yaml

```diff
@@ -2,24 +2,35 @@ name: google_fonts
 description: A package to include fonts from fonts.google.com in your Flutter app.
 version: 1.1.1
```

Comment:
version should be `2.0.0-nullsafety` or `2.0.0-nullsafety.0`

Reply:
Done.